src/main/java/frc/robot/constants/IntakeConstants.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.constants;
+
+public final class IntakeConstants {
+  
+  public static final int left_MOTOR_ID = -1;
+
+  public static final int right_MOTOR_ID = -1;
+
+  public static final double intake_SPEED = 0.5;
+
+  public static final double outtake_SPEED = 0.5;
+
+  public static final double rotate_SPEED = 0.5;
+
+  public static final double spin_SPEED = 0.5;
+
+  public static final double intake_CONTINUOUS_SPEED = 0.5;
+
+  public static final double outtake_CONTINUOUS_SPEED = 0.5;
```

Comment:
We don't use underscores, just camelCase, so only the first letter of each word is capitalized. And k before constants.